rllib/examples/actions/custom_action_distribution.py

```diff
@@ -0,0 +1,118 @@
+"""Example on how to define and run an experiment with a custom action distribution.
+
+The example uses an additional `temperature` parameter on top of the built-in
```

Comment:
Great example of how to introduce temperature into the action sampling. Could we also show how to decay this temperature. Temperature decay over the course of training is a common practice in RL.